easybuild/easyconfigs/s/SIONlib/SIONlib-1.6.1-foss-2016a-tools.eb

```diff
@@ -0,0 +1,41 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2016 Juelich Supercomputing Centre, Germany
+# Authors::   Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SIONlib'
+version = '1.6.1'
+versionsuffix = '-tools'
```

Comment:
@geimer: can you include a short comment above this line the highlight the difference that warrants this version suffix, since it's easy to miss?